component-classes/vite.config.js

```diff
@@ -15,6 +16,14 @@ export default () => ({
       insertTypesEntry: true,
       outputDir: '../dist',
     }),
+    {
+      name: 'build-script',
+      buildStart() {
```

Comment:
This can also be added to the build step as script. If we want to keep it in the same place

Reply:
Do you mean to have the `build:cc` command look something like this instead?

```
    "build:cc": "cd component-classes && rm -rf *.d.ts && tsc && node ./scripts/updateClassesToPurge.js"
``` 

If so, that does sound like a good idea to me. Then we'd show everything that's happening in the `build:cc` script in one place.